packages/slice-machine/components/FormFields/Input.tsx

```diff
@@ -2,9 +2,9 @@ import { Field, FieldInputProps, FieldMetaProps } from "formik";
 import { Box, Label, Input, Text, ThemeUIStyleObject } from "theme-ui";
 
 const getFieldStyles = (
-  isError: boolean,
-  isDisabled: boolean,
-  isWarning: boolean
+  isError: boolean | undefined,
```

Comment:
OPT: Are those distinctive conditions or could they be fused with a Typescript enum ?

Reply:
Good spot, actually because i've put Boolean(meta.error), it can only be a boolean so ive just removed the undefined